.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to GLPI
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+Please note current repository is about GLPI core only. All related plugins are the responsibility of their respective owners and all requests must be done on their own systems.
```

Comment:
`Please note current repository is about GLPI core only.` -> `Please note that the current repository is containing GLPI core only.` ?